docs/README.md

```diff
@@ -2541,7 +2528,42 @@ npm init
 npm install --save @boostercloud/framework-provider-aws-infrastructure
 ```
 
-The implementation of `InfrastructureRocket` might vary from one provider to the other, but in AWS it only requires two functions:
+The schema of an *Infrastructure Rocket* project should look something like this:
+
+```text
+rocket-your-rocket-name-aws-infrastructure
+├── package.json
+├── src
+    ├── lambdas
+    ├── index.ts
+    └── your-main-class.ts
+
+```
+
+In `<your-main-class>.ts` is where you define the two functions that AWS requires:
```

Comment:
`your-main-cass` can be named as you want and this is where we define the `mountStack` and `unmount` methods.